### ISSUE TITLE
Rewrite :focus state for the input

### ISSUE THREAD

**Issue by martinjinda** [snapappointments/bootstrap-select#2356]
I would like to change default focus state color so it is corresponding with other inputs, but I don't find any way how to change this line, why is there important everywhere, is that really necessary? 

https://github.com/snapappointments/bootstrap-select/blob/a5b075219d3a87336c046a5a0403023eed8f4c6e/sass/bootstrap-select.scss#L127

Preview of my focus with this 5px width border from bootstrap-select:
![Screenshot 2019-11-21 at 14 56 43](https://user-images.githubusercontent.com/9948632/69344156-32b75400-0c6f-11ea-906d-29249ae47fa3.png)

**Comment by NicolasCARPi** [comment]
I agree that `!important` should be used sparingly. @caseyjhol is there a good reason to have `!important` in the css?
